htdocs/compta/facture/card-rec.php

```diff
@@ -637,7 +637,7 @@
 				$info_bits |= 0x01;
 			}
 
-			if ($usercanproductignorepricemin && (!empty($price_min) && (price2num($pu_ht) * (1 - price2num($remise_percent) / 100) < price2num($price_min)))) {
+			if ($usercanproductignorepricemin && (!empty($price_min) && (price2num($pu_ht) * (1 - price2num((float) $remise_percent) / 100) < price2num($price_min)))) {
```

Comment:
What was the value of $remise_percent that cause a fatal error ?
price2num should be able ta accept a string to convert it into a float. And value may be a string that can't be converted by (float) like '1 234,56'.

Reply:
Although a discount percentage value I don't think it will reach to "1 234.56", you are right, and I have generated a more appropriate solution